keras_core/backend/jax/layer.py

```diff
@@ -1,4 +1,18 @@
+from keras_core.backend.jax import distribution
+
+
 class JaxLayer:
     def _post_build(self):
         """Can be overriden to perform post-build actions."""
-        pass
+        if distribution.get_distribution() is None:
+            return
+
+        distribute = distribution.get_distribution()
+        # Swap both trainable/non-trainable weights,
```

Comment:
We should do this in `Variable._initialize()`, right? It's already doable for data parallel, and for model parallel it will be doable when we have name scopes (imminently)

Reply:
Ack. Moved this to jax.variable.